JUSTFILE

```diff
@@ -105,8 +110,71 @@ gen-report:
   allure generate tests/allure/results --clean -o tests/allure/reports && cp tests/allure/reports/history/* tests/allure/results/history/. && allure open tests/allure/reports
 
 run-tests:
-  helm test cloudbeat-tests --namespace kube-system --logs
+  helm test {{TESTS_RELEASE}} --namespace kube-system
 
 build-load-run-tests: build-pytest-docker load-pytest-kind run-tests
 
-prepare-local-helm-cluster: create-kind-cluster build-cloudbeat load-cloudbeat-image deploy-local-tests-helm
\ No newline at end of file
+delete-local-helm-cluster:
+  kind delete cluster --name kind-mono
+
+cleanup-create-local-helm-cluster target: delete-local-helm-cluster create-kind-cluster build-cloudbeat load-cloudbeat-image
+  just deploy-local-tests-helm {{target}}
+
+test-pod-status:
+  #!/usr/bin/env sh
+
+  if [ ${STATUS=`kubectl get pod -n kube-system test-pod-v1 --template {{{{.status.phase}}`} ]; then
+    echo $STATUS
+  else
+    echo {{POD_STATUS_UNKNOWN}}
+  fi
+
```

Comment:
I think we need to avoid scripting inside justfile. The better way is to create helper utils lib and execute scripts from the lib.

Reply:
@gurevichdmitry @yashtewari  please note we already have it.These two dirs `scripts/ `& `.ci/scripts` .
 If you could please either generate a task and comment a Todo to the justfile, or move the bahs logic into one of these two dirs.

Reply:
@yashtewari if u can comment a Todo with task number https://github.com/elastic/security-team/issues/4291